easybuild/easyconfigs/a/ARP_wARP/ARP_wARP-7.6.eb

```diff
@@ -0,0 +1,78 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Adam Mazur
+# Research IT
+# Biozentrum, University of Basel
+# Modified by:
+# Adam Huffman
+# The Francis Crick Institute
+
+easyblock='Tarball'
+
+name = 'ARP_wARP'
+version = '7.6'
+
+homepage = 'http://www.embl-hamburg.de/ARP/'
+description = """ Crystallographic macromolecular model building. Builds proteins, RNA/DNA, 
+ secondary structure, side chains, loops, solvent and ligands. Intended to be used in 
+ conjunction with CCP4. """ 
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+# Need to agree to the licence for the download, either from the CCP4 site or from
+# http://www.embl-hamburg.de/ARP/download_request7.6.html
+sources = ['%(namelower)s_%(version)s.tar.gz']
+
+ccp4dir = '%(namelower)s-%(version)s'
+arpwarpdir = 'arp_warp_7.6'
+
+postinstallcmds = [
+    'touch $HOME/.agree2ccp4v6',
+    'cd %(installdir)s/' + ccp4dir + ' && ./BINARY.setup',
+    'cd %(installdir)s/' + arpwarpdir + ' && ./install.sh ccp4=%(installdir)s/' + ccp4dir,
+]
+
+sanity_check_paths = {
+    'files': [ccp4dir + '/bin/ccp4.setup-sh'], # this file is created after ./BINARY.setup postinstall cmd
+    'dirs': [ccp4dir, arpwarpdir],
+}
+
+# module definition is based on installdir/ccp4-6.5/include/ccp4.setup-module
+
+modextrapaths = {
+    'PATH': '%s/bin:$root/%s/etc:$root/%s/bin/bin-x86_64-Linux:$root/%s/share/xia2/Applications' % (ccp4dir, ccp4dir, arpwarpdir, ccp4dir),
+    'MANPATH': '%s/share/man' % ccp4dir
+}
+ 
+modextravars = {
+    'CCP4_MASTER': '$root',
+    'CCP4': '$root/%s' % ccp4dir,
+    'CCP4I_TCLTK': '$root/%s/bin' % ccp4dir,
+    'BALBES_ROOT': '$root/BALBES',
+
+    'GFORTRAN_UNBUFFERED_PRECONNECTED': '1',
+    'CBIN': '$root/%s/bin' % ccp4dir,
+    'CLIB': '$root/%s/lib' % ccp4dir,
+    'CCP4I_TOP': '$root/%s/share/ccp4i' % ccp4dir,
+    'CCP4_OPEN': 'UNKNOWN',
+    'MMCIFDIC': '$root/%s/lib/ccp4/cif_mmdic.lib' % ccp4dir,
+    'CRANK': '$root/%s/share/ccp4i/crank' % ccp4dir,
+    'CLIBD_MON': '$root/%s/lib/data/monomers/' % ccp4dir,
+    'CCP4_HELPDIR': '$root/%s/help/' % ccp4dir,
+
+    'CETC' : '$root/%s/etc' % ccp4dir,
+    'CEXAM' : '$root/%s/examples' % ccp4dir,
+    'CHTML' : '$root/%s/html' % ccp4dir,
+    'CINCL' : '$root/%s/include' % ccp4dir,
+    'CLIBD' : '$root/%s/lib/data' % ccp4dir,
+
+    'warpbin': '$root/%s/bin/bin-x86_64-Linux' % arpwarpdir,
+    'warpdoc': '$root/%s/manual' % arpwarpdir
+}
+
+modtclfooter = """set ccp4_scr "/tmp/$env(USER)" setenv CCP4_SCR ${ccp4_scr}
+if { ! [ file exists $ccp4_scr ] } {
+    file mkdir $ccp4_scr
```

Comment:
is this really necessary?

wouldn't it be better to patch to the tool to do this instead if `$CCP4_SCR` doesn't exist (and submit the patch upstream)?